src/main.ts

```diff
@@ -267,12 +267,13 @@ async function handleFindTrack(): Promise<void> {
 async function findTrack(trackId: number): Promise<void> {
   const newTrack = dataset!.buildTrack(trackId);
 
-  if (newTrack.length() < 1) {
+  if (newTrack.times.length < 1) {
```

Comment:
why this change? I believe it is still safer and more maintainable to call `newTrack.length()`

Reply:
Changed, thank you!